src/menu/index.js

```diff
@@ -44,7 +55,8 @@ class Menu {
     );
 
     const scriptNames = Object.keys(GlobalConfig.getScripts());
-    const choices = [...scriptNames, 'help', 'quit'];
+    const decoratedScriptNames = getDocumentedChoices({ choices: scriptNames });
```

Comment:
nit:
- [ ] remove `decoratedScriptNames` variable and just spread `getDocumentedChoices` directly into choices array